Content.Shared/Delivery/DeliveryModifierSystem.cs

```diff
@@ -148,6 +235,27 @@ private void UpdatePriorty(float frameTime)
             }
         }
     }
+
+    private void UpdateBomb(float frameTime)
+    {
+        var bombQuery = EntityQueryEnumerator<DeliveryBombComponent, PrimedDeliveryBombComponent>();
```

Comment:
Swap the order of these around, so the game enumerates over `PrimedDeliveryBombComponent` as primary component (as those will be less numerous).